server/src/main/java/org/opensearch/common/network/NetworkModule.java

```diff
@@ -89,6 +92,7 @@ public final class NetworkModule {
 
     public static final String TRANSPORT_TYPE_KEY = "transport.type";
     public static final String HTTP_TYPE_KEY = "http.type";
+    public static final String AUX_TRANSPORT_TYPE_KEY = "aux_transport.type";
```

Comment:
I think we should definitely go with list of aux transports right away:

```suggestion
    public static final String AUX_TRANSPORT_TYPE_KEY = "aux_transport.types";
```